app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -32,6 +32,8 @@ You need [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to
 
 You do not need QTS to [teach in further education](/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers) or to [teach in early years](/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers).
 
+You do not need a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce) to be a qualified teacher in England.
```

Comment:
Worth repeating what is on the PGCE page (as below) or similar? Sounds a bit harsh here, particularly when a lot of people do courses with PGCE, and it is such a big keyword?


```suggestion
You do not need a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce) to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.
```